packages/katello/katello/katello-remove

```diff
@@ -141,38 +164,49 @@ CONTENT.map! do |dir|
 end
 
 def remove
-  puts 'Stopping services'
-  `katello-service stop`
+  puts "\nStopping services".yellow
+  `foreman-maintain service stop`
 
-  puts 'Removing RPMs'
+  puts "\nRemoving RPMs".yellow
   packages = []
   RPMS.each do |rpm|
     rpmpackages = `rpm -qa | grep "#{rpm}"`
     packages += rpmpackages.lines.map(&:chomp)
   end
   `yum erase -y #{packages.join(' ')} > /dev/null 2>&1`
 
-  puts 'Cleaning up configuration files'
-  FileUtils.rm_rf CONFIG_FILES
-
-  puts 'Cleaning up log files'
-  # logs
-  FileUtils.rm_rf LOG_FILES
-
-  puts 'Cleaning up certs'
-  # pulp cert stuff
-  FileUtils.rm_rf CERT_FILES
-
-  puts 'Cleaning up content'
-  # content
-  FileUtils.rm_rf CONTENT
+  puts "\nCleaning up configuration files".yellow
+  FileUtils.rm_r Dir.glob(CONFIG_FILES)
+
+  puts "\nCleaning up log files".yellow
+  # Logs
+  FileUtils.rm_r Dir.glob(LOG_FILES)
+
+  puts "\nCleaning up certs".yellow
+  # Cert stuff
+  FileUtils.rm_r Dir.glob(CERT_FILES)
+
+  puts "\nCleaning up content".yellow
+  # Content
+  FileUtils.rm_r Dir.glob(CONTENT)
+
+  puts "\nRemoving Databases and Pulp storage on Filesystem".yellow
+  # Check and see if MOUNTS are mounted filesystems.
+  MOUNTS.each do |mount|
+    `mountpoint -q #{mount}`
+    if $?.success?
+      puts "\n #{mount} is part of a mounted filesystem, skipping. You will need to remove #{mount} manually.".red
```

Comment:
Should this remove `Dir.glob(File.join(mount, '*'))`? Probably good to ensure it's not `/*` just to be sure though.